src/main/groovy/org/jenkinsci/gradle/plugins/jpi/JpiManifest.groovy

```diff
@@ -137,7 +201,6 @@ class JpiManifest extends Manifest {
         YesNoMaybe.YES
     }
 
-    static Map<String, ?> attributesToMap(Attributes attributes) {
-        attributes.collectEntries { k, v -> [k.toString(), v] } as Map<String, ?>
+    static class JpiAttributes extends HashMap<String, Object> implements Attributes {
```

Comment:
I think it would be better to use a `LinkedHashMap` to have a repeatable order of elements.